test/e2e_node/runner/run_e2e.go

```diff
@@ -76,18 +97,50 @@ func main() {
 		return
 	}
 
-	if *hosts == "" && *images == "" {
-		glog.Fatalf("Must specify one of --images or --hosts flag.")
+	if *hosts == "" && *imageConfigFile == "" && *images == "" {
+		glog.Fatalf("Must specify one of --image-config-file, --hosts, --images.")
 	}
-	if *images != "" && *zone == "" {
-		glog.Fatal("Must specify --zone flag")
+	gceImages := &ImageConfig{
+		Images: make(map[string]GCEImage),
 	}
+	if *imageConfigFile != "" {
+		// parse images
+		imageConfigData, err := ioutil.ReadFile(*imageConfigFile)
+		if err != nil {
+			glog.Fatalf("Could not read image config file provided: %v", err)
+		}
+		err = yaml.Unmarshal(imageConfigData, gceImages)
+		if err != nil {
+			glog.Fatalf("Could not parse image config file: %v", err)
+		}
+	}
+
+	// Allow users to specify additional images via cli flags for local testing
+	// convenience; merge in with config file
 	if *images != "" {
 		if *imageProject == "" {
-			glog.Fatal("Must specify --image-project flag")
+			glog.Fatal("Must specify --image-project if you specify --images")
 		}
+		cliImages := strings.Split(*images, ",")
+		for _, img := range cliImages {
+			gceImages.Images[img] = GCEImage{
+				Image:   img,
+				Project: *imageProject,
+			}
+		}
+	}
+
+	if len(gceImages.Images) == 0 && *zone == "" {
```

Comment:
Oops, original condition was actually `*images != ""` so that should be `len != 0`, fixing...